README.md

```diff
@@ -50,14 +53,16 @@ Signal -> Chats -> Backups -> Local Backup
 5. Run the script in this repo over the XML file:
 
 ```
-python3 sms-mms-import-to-signal.py sms-20220611130220.xml bits/
+python3 sms-mms-import-to-signal.py sms-20220611130220.xml bits
+python3 --input sms-mms-import-to-signal.py --output sms-20220611130220.xml bits -m -v 
```

Comment:
I think the --input flag is probably meant to go later in the command here

Reply:
yeah this command is all kinds of wrong, good pickup